domain/domain.go

```diff
@@ -1390,6 +1390,8 @@ func (do *Domain) updateStatsWorker(ctx sessionctx.Context, owner owner.Manager)
 	gcStatsTicker := time.NewTicker(100 * lease)
 	dumpFeedbackTicker := time.NewTicker(200 * lease)
 	loadFeedbackTicker := time.NewTicker(5 * lease)
+	//dumpColStatsUsageTicker := time.NewTicker(20 * lease)
```

Comment:
Restore this line and remove the next line?

Reply:
Restored.